documentation/staging/content/userguide/managing-domains/domain-lifecycle/introspection.md

```diff
@@ -76,7 +85,7 @@ when `spec.logHome` is configured and `spec.logHomeEnabled` is true.
 
 ### Introspection use cases
 
-#### Adding clusters or Managed Servers to the WebLogic domain configuration
+#### Adding clusters or Managed Servers to a "Domain in PV" configuration
```

Comment:
Since the "Domain in PV" acronym is explained in the following paragraph, should the heading say something like "Adding clusters or Managed Servers to the WebLogic domain configuration with domain home in persistent volume"?

Reply:
The "Domain in PV" phrase is common throughout the WKO docs, and so including it makes the title more clear/precise IMO. We might want to remove the quotes though...
I've no strong opinion, we can leave this up to @rosemarymarano

Reply:
I think Domain in PV is ok in the title, however, you don't need the quotation marks there or in the follow up paragraph.

Reply:
OK - I've removed the quotation marks.